recipes/correct-ts-specifiers/src/map-imports.test.ts

```diff
@@ -6,19 +6,25 @@ import { dExts } from './exts.ts';
 import type { FSAbsolutePath } from './index.d.ts';
 
 
-type Logger = typeof import('./logger.ts').logger;
+type Logger = typeof import('@nodejs/utils/logger').default;
 type MapImports = typeof import('./map-imports.ts').mapImports;
 
 describe('Map Imports', { concurrency: true }, () => {
 	const originatingFilePath = fileURLToPath(import.meta.resolve('./test.ts')) as FSAbsolutePath;
-	let mock__log: Mock<Logger>['mock'];
+	let mock__log: Mock<Logger['error']>['mock'];
 	let mapImports: MapImports;
 
 	before(async () => {
-		const logger = mock.fn<Logger>();
-		({ mock: mock__log } = logger);
-		mock.module('./logger.ts', {
-			namedExports: { logger },
+		const errorMock = mock.fn<Logger['error']>();
+		const logger = {
+			error: errorMock,
+			warn: mock.fn<Logger['warn']>(),
+			info: mock.fn<Logger['info']>(),
+			debug: mock.fn<Logger['debug']>()
+		};
+		mock__log = errorMock.mock;
+		mock.module('@nodejs/utils/logger', {
+			defaultExport: logger,
```

Comment:
nit: I think it's not necessary to type all of the methods (they aren't called in the test).